demo/js/tree-view.js

```diff
@@ -22,21 +17,15 @@ window.onload = function() {
     grid = new Hypergrid('div#tree-example');
     grid.setData(treeData);
 
-    // Install the sorter API (optional).
-    new Hypersorter(grid, { // eslint-disable-line no-new
-        Column: fin.Hypergrid.behaviors.Column
-    });
-
-    // Install the drill-down API (optional).
-    var dataModel = grid.behavior.dataModel,
-        dataModelPrototype = Object.getPrototypeOf(dataModel);
-    drillDown.mixInTo(dataModelPrototype);
+    grid.installPlugins([
+        Hypergrid.drillDown, // simple API install (plain object with `install` method) but no `name` defined so no ref is saved
+        Hypergrid.rowById, // ditto
+        Hypergrid.Hyperfilter, // object API instantiation; `$$CLASS_NAME` defined so ref saved in `grid.plugin.hyperfilter`
+        Hypergrid.Hypersorter, // object API instantiation to grid.plugin; no `name` or `$$CLASS_NAME` defined so no ref saved
+        ['treeviewAPI', Hypergrid.TreeView, options] // object API instantiation with one arg; [0] overrides any defined name so ref saved in `grid.plugin.treeViewAPI`
```

Comment:
Maybe an object here? I usually use objects. The only reason I went with an array here is it "mimics" the constructor args. Alternative would look like:
```javascript
        { name: 'treeviewAPI', API: Hypergrid.TreeView, options, args: options }
```